app/models/account.rb

```diff
@@ -0,0 +1,10 @@
+class Account
+  include Braque::Model
+  self.api_root = Rails.application.config_for(:echo_api)['url']
+  self.api_token = Rails.application.config_for(:echo_api)['token']
```

Comment:
[Braque](https://github.com/dylanfareed/braque) provides a thin wrapper around [hyperclient](https://github.com/codegram/hyperclient). This sets up the `Account` model with remote API details and basic model attributes